data/yaml/venues/multi3generation.yaml

```diff
@@ -0,0 +1,3 @@
+acronym: Multi3Generation
+name: The 1st International Workshop on Multilingual, Multimodal and Multitask Language
```

Comment:
The name in the venues file is the name of the workshop, not a meeting of the workshop (so no "1st")

Reply:
Done.